spec/datetimeformat.html

```diff
@@ -58,6 +58,11 @@ <h1>Abstract Operations For DateTimeFormat Objects</h1>
           <td>`"second"`</td>
           <td>`"2-digit"`, `"numeric"`</td>
         </tr>
+        <tr>
+          <td>[[FractionalSecondDigits]]</td>
+          <td>`"fractionalSecondDigits"`</td>
+          <td>`"1"`, `"2"`, `"3"`</td>
```

Comment:
Why only 1, 2, and 3?  Temporal goes to 9 digits of precision.  Or are you intending to defer the 4-9 digits of precision into the Temporal proposal?

Also, why are these strings?

Reply:
1. We discussed this up to 3 digits precision issue in TC39 before (July?) The reason is currently 
ECMA262 is representing all the time information internally in milliseconds but not smaller unit.
https://ecma-international.org/ecma-262/#sec-time-values-and-time-range
```
Time is measured in ECMAScript as milliseconds since midnight at the beginning ...
```
See https://ecma-international.org/ecma-262/#sec-maketime
https://ecma-international.org/ecma-262/#sec-timeclip
https://ecma-international.org/ecma-262/#sec-date-time-string-format

And since currently the Intl.DateTimeFormat.prototype.format* only accept Date object, and there are no way to receive more precise time information than milliseconds, allowing fractional second digits  at this point more than 3 will be 1) difficult to implement and 2) no practical use, 3) unnecessary. In the TC39 July meeting we also agree that it could be later easily extend to accepting more digits if other proposal define mechanism to provide more precious time information to feed into Intl.DateTimeFormat , but that will be out of the scope of this proposal.

So, in summary, yes, we should defer that for Temporal proposal to define, or a even later PR once both got into Stage 3.

About String type: I will change it back to number with 0, 1, 2, 3.